war/src/main/less/pluginSetupWizard.less

```diff
@@ -342,7 +342,7 @@
 
 		            &:hover{
 		              background: rgba(100, 200, 255, 0.1);
-		              box-shadow:inset 0 200px 200px -200px rgba(100, 200, 255, 0.33),inset 0 0 0 1px rgba(100, 200, 255, 0.33);
+                      box-shadow:inset 0 400px 400px -100px rgba(100, 200, 255, 0.33), inset 0 0 0 1px rgba(100, 200, 255, 0.33);
```

Comment:
.. there was an actual change here, not just whitespace: made the box shadow overlay expand farther, as the box is larger when it has dependencies expanded